### ISSUE TITLE
URLs badly formed

### ISSUE THREAD

**Issue by sckott** [ropensci/codemetar#48]
in the latest installed from github (at this commit https://github.com/ropensci/codemetar/tree/c371dbbeb080992a1cfd596cb0354c983d6f285e ) i was getting bad URLs, at least these two. Diff from running `codemetar::write_codemeta()` in `fulltext` base dir

```diff
-  "releaseNotes": "https://github.com/ropensci/fulltext/blob/master/NEWS.md",
-  "readme": "https://github.com/ropensci/fulltext/blob/master/README.md",
-  "fileSize": "3775.796KB"
+  "releaseNotes": "https://github.com/ropensci/fulltext.git/blob/master/NEWS.md",
+  "readme": "https://github.com/ropensci/fulltext.git/blob/master/README.md",
+  "fileSize": "3776.894KB",
+  "developmentStatus": "active"
 }
```
Unless the `.git` is supposed to be there for some reason?

**Comment by cboettig** [comment]
Whoops, probably left over from when we parse the git remote address.  Should now be fixed in  194cf7e .

**Comment by sckott** [comment]
works, thanks
